packages/components/src/tab-panel/index.js

```diff
@@ -63,6 +64,13 @@ class TabPanel extends Component {
 
 		const selectedTab = find( tabs, { name: selected } );
 		const selectedId = instanceId + '-' + selectedTab.name;
+		
+		deprecated( 'Tab Panel child function argument used as string', {
+			alternative: "Argument is now an object, access the name property directly.",
```

Comment:
I think the failing build might be related to the use of double quote here. We should use the single quote.

Reply:
Good catch, thanks!